### ISSUE TITLE
Error on using maps

### ISSUE THREAD

**Issue by Tritlo** [google/proto-lens#11]
I define a file `LinkParser.proto`
```
syntax = "proto3";

message LinkParseResult {
  string title = 1;
  map<string, string> og = 2;
  repeated string imgs = 3;
}
```

When I run `stack build`, and also when I run:
```
protoc --plugin=protoc-gen-haskell=`which proto-lens-protoc` --haskell_out . LinkParser.proto
```

manually,  I get the following error:

```
proto-lens-protoc: definedFieldType: Field type .LinkParseResult.OgEntry not found in environment.
--haskell_out: protoc-gen-haskell: Plugin failed with status code 1.
```

Happens on both `libprotoc 3.0.0` and `libprotoc 2.6.1` (where the latter used the "equivalent syntax" mentioned in the `protoc` docs).

**Comment by judah** [comment]
Thanks for the report!  Looks like map fields don't work when there's no "package" declaration as in:
https://github.com/google/proto-lens/blob/master/proto-lens-tests/tests/proto3.proto

I'll look into a fix; in the meantime you should be able to work around it by defining a "package" for the .proto file.

**Comment by Tritlo** [comment]
Thanks for the quick reply; that workaround did the trick.

**Comment by Tritlo** [comment]
Great! Is there an ETA on the next hackage release?

**Comment by judah** [comment]
The fix has been released on Hackage as v0.1.0.1.
